lib/alerts/cache/fetcher.ex

```diff
@@ -48,7 +48,10 @@ defmodule Alerts.Cache.Fetcher do
   def handle_info(:fetch, {update_fn, api_mfa, repeat_ms} = state) do
     case api_result(api_mfa) do
       %{data: data} ->
-        alerts = Enum.map(data, &Parser.Alert.parse/1)
+        alerts =
+          data
+          |> Enum.map(&Parser.Alert.parse/1)
+          |> Enum.reject(&suppressed_alert?/1)
```

Comment:
kinda feels not worth it to parse alerts before rejecting, what're your thoughts on discarding the bad alerts before parsing?